### ISSUE TITLE
Most of the FEA structural analysis is poor

### ISSUE THREAD

**Issue by moorepants** [moorepants/eme185#43]
The students don't seem to have learned good practices for modeling choices in FEA. They don't create free body diagrams and look at individual components. They often incorrectly use constraints and load types. They rarely report their modeling assumptions. Rarely talk about mesh and they never use simpler FEA models, always trying to do things with the most complex geometry.

The learning they are doing on their own for this doesn't really bring things up to par. I'd much rather see them make a simple analytic model with 150A skills to make simple conclusions instead of erroneous conclusions with bad FEA models.

**Comment by moorepants** [comment]
It would be nice to have a lesson on this or a resource page that shows them how to do this.

**Comment by moorepants** [comment]
@swkresie Since Naveen is leading the microcontroller lesson, Ian the 3D printing one, would you like to lead this. It would then be you and my primary responsibility and Ian and Naveen can participate too as desired.

**Comment by swkresie** [comment]
Yes, I can lead the FEA module. Can you send me the teaching materials you
currently have on this?

How much in class time  is allocated  for the module?

On Feb 7, 2018 12:55 PM, "Jason K. Moore" <notifications@github.com> wrote:

@swkresie <https://github.com/swkresie> Since Naveen is leading the
microcontroller lesson, Ian the 3D printing one, would you like to lead
this. It would then be you and my primary responsibility and Ian and Naveen
can participate too as desired.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/moorepants/eme185/issues/43#issuecomment-363907318>,
or mute
the thread

**Comment by moorepants** [comment]
I dropped some files in `EME185/Previous Class Resources/materials-from-jiancheng-liu`. These are the tutorials made with an NSF grant on the topic and he has waived any copyright for us to use them. i'll dig up the modules from EME 150A too.

Timing should be tied to 50 minute blocks. We can use two or three in one studio and we could even spread to two studios if needed.

**Comment by moorepants** [comment]
I forgot that the last studio section will be dedicated to the lightning talks. So we only have 1 full studio session that could be used for this.

**Comment by moorepants** [comment]
I added a folder called "materials-from-matthew-lefort" that has some modules that were developed for eme 150a. I think these are a bit more in scope.

**Comment by moorepants** [comment]
Results of Scott and my's meeting today:

# What might a student want to (or should) learn?

- when is FEA the right tool for the job?
- how to model (FBD > ID crit > create simple geom > define FEA model)
- how to present results
- common pitfalls
- details of operating specific software
- how to present modeling assumptions
how to identify critical parts/elements to examine
- what is all the FEA jargon
- defining geom, constraints, loads, mesh, nodes and consequences for each

# Ranked Learning Objectives

Students will be able to:

1. present structural FEA results in a report that fully defines the model, assumptions, methods, and highlights key finds from results
2. identify when FEA is an appropriate analysis tool for their design
3. create a fininte element mdel of a structural element using a balid modeling procedure
..1. identify the critical structural components of their design given a FBD
..2. create a simplified structure of a critical part to model using FEA
..3. Define geometry, constraints, loads, mesh, nodes and consequences for each.
..4. translate a structure and FBD into constraints, loads, materials, and mesh in solidworks.
4. recall definitions of FEA terms and jargon
5. identify common pitfalls in existing models and results

# Class

## [30 mins] Introduction
- what is FEA
- terms and jargon
- identifying when FEA is right tool

## [70 mins] Modeling

## [50 mins] Presenting results

**Comment by swkresie** [comment]
Will we be doing this lesson for Prof. Velinsky's Tuesday 4-7PM section as
well? If so, it will conflict with Ian's Prelim. Design Review meeting time
with the sterilize team.


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#m_4987854111148171323_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by moorepants** [comment]
No we will not be doing this for Prof. Velinsky's section.
